README.md

```diff
@@ -78,11 +78,24 @@ pytest -s -v tests/
 
 
 ## Release
-Releases are published automatically by GitHub Actions when a tag (v**) is pushed to GitHub.
-- Make sure you tag the correct commit
-- Pushing a tag to GitHub requires maintainers/admin privileges
+Releases are automatically released via GitHub Actions when a new tag `v**` is pushed to GitHub. 
 
-```commandline
+```shell
 git tag v1.0 <some-commit-hash>
-git push upstream v1.0
-```
\ No newline at end of file
+git push origin v1.0
+```
+
+**Note:** pushing a tag to GitHub requires maintainers/admin privileges.
+
+## Contributing
+Contributions are welcomed! please create a fork and open a pull request to submit your changes,
+
+Contributing requires installation from source:
+1. Clone scylla-api-client repository
+2. Install dev-tools `pip install -r dev-requirements.txt`
```

Comment:
`dev-tools` looks like a name of a package.
Better change to "development tools".